### DISCUSSION TITLE
Screen.sleepTimeout 기능 추가 건의

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#925] [5. Q&A]
몇몇 게임의 경우 슬립타임아웃 시간을 조절하고 싶은데, 현재는 사용시 예외가 발생하고 있습니다.

앱정책은 SleepTimeout.SystemSetting으로 맞춰져있을것 같은데, 
월드종료시 강제로 시스템셋팅으로 설정하게 한다면 게임내에서만 적용될테니 큰문제가 없다면 지원되었으면 합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 건의 주신 내용 내부에서 긍정적으로 검토해보겠습니다.
감사합니다.
